yt/frontends/ramses/definitions.py

```diff
@@ -41,11 +41,6 @@ def ramses_header(hvals):
                  ('timing', 5, 'd'),
                  ('mass_sph', 1, 'd') )
     yield next_set
-    tree_header = ( ('headl', hvals['nlevelmax'] * hvals['ncpu'], 'i'),
```

Comment:
We don't need this anymore?

Reply:
It's not used anywhere so we can skip them.